spec.bs

```diff
@@ -4580,6 +4581,33 @@ Width and height of an ad.
 
 </dl>
 
+<h3 id=direct-from-seller-signals-header>Direct from seller signals</h3>
+
+A <dfn>direct from seller signals key</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="direct from seller signals key">
+  : <dfn>seller</dfn>
+  :: An [=origin=]. Matches the origin that served the captured [:Ad-Auction-Signals:] header.
+  : <dfn>ad slot</dfn>
+  :: A [=string=]. Matches the `adSlot` key of the JSON dictionaries in the top-level array of the
+    [:Ad-Auction-Signals:] value.
+</dl>
+

```

Comment:
Should we bring back the `<h3 id=direct-from-sellers-signals-struct>Direct from seller signals</h3>` here?

Reply:
SGTM